client/src/components/Invitation.js

```diff
@@ -75,18 +75,81 @@ export default function FormDialog() {
 
   /*push emails to emailList*/
   const addEmail = () => {
-    // if (emailList.findIndex((email) => email == email) < 0)
+    if (!email) return;
     if (!emailList.includes(email)) {
       setEmailList([...emailList, email]);
-      setEmail("");
-    } else return;
+      setEmail('');
+    } else {
+      return;
+    }
   };
 
-  const submitEmail = async () => {
+  //handles email invitations
+  // maybe use a Material UI loading component?
+  const submitInvite = async () => {
     handleClose();
 
-    //need to grab current user id and email
-    sendInvite("5fad63358f96786e507a0b74", emailList);
+    let goodEmails = [];
+    let badEmails = [];
```

Comment:
We could probably have more clear names for these, maybe successfullySent and unsuccessful or something?